aws-lambdas/iac/cdk/common/constants/index.ts

```diff
@@ -2,4 +2,3 @@ export * from './lambda-role-policy';
 export * from './sqs-role-policy';
 export * from './sns-role-policy';
 export * from './iam-role-policy';
-
```

Comment:
Something is off here if the last empty line is being removed. pre-commit hooks will add this back I believe.

Reply:
it did not